routes/carts.js

```diff
@@ -0,0 +1,109 @@
+const express = require('express');
+const router = express.Router();
+const knex = require('../database');
+
+router.route('/:user_id').get((req, res) => {
+  knex
+    .raw('select * from users where id = ?', [req.params.user_id])
```

Comment:
Getting the user isn't really necessary. But if you wanted to do this, you'll have to move the two .thens into the first .then and nest all that work. The reason being that if there's no user that matches this id, it'll send the 404 but still move on to the .thens below. This could cause multiple res.sends to fire which will cause an error on express.